lib/src/radio-group/radio-group.component.html

```diff
@@ -1,15 +1,35 @@
+<label *ngIf="label" class="radio-group-label"
+    ><b>{{ label }}</b></label
+>
 <div [attr.id]="id" *ngIf="list" class="radio-group {{ className }}" [class.disabled]="disabled">
-    <label *ngIf="label" class="radio-group-label"
-        ><b>{{ label }}</b></label
-    >
-    <ng-container *ngFor="let item of displayList; index as i">
-        <div class="{{ item.className }}" (click)="handleItemOnClick(item)">
-            <div class="radio-item d-flex">
-                <label class="radio-label flex-fill" [attr.for]="item.id">{{ item.optionItem.label }}</label>
-                <input class="radio-input" type="radio" [name]="id" [id]="item.id" [disabled]="disabled" [defaultChecked]="item.selected" />
-
-                <span class="radio-indicator"></span>
+    <div *ngIf="inline; else noninline" class="row">
+        <ng-container *ngFor="let item of displayList; index as i">
+            <div class="col">
+                <ng-container *ngTemplateOutlet="readioitem; context: { $implicit: item }"></ng-container>
             </div>
-        </div>
-    </ng-container>
+        </ng-container>
+    </div>
+
+    <ng-template #noninline>
+        <ng-container *ngFor="let item of displayList; index as i">
+            <ng-container *ngTemplateOutlet="readioitem; context: { $implicit: item }"></ng-container>
+        </ng-container>
+    </ng-template>
 </div>
+
+<ng-template #readioitem let-item>
+    <div class="{{ item.className }}" (click)="!(item.optionItem.disabled || disabled) ? handleItemOnClick(item) : null">
+        <input
+            [name]="id"
+            [id]="item.id"
+            [defaultChecked]="item.selected"
+            [disabled]="item.optionItem.disabled || disabled"
+            type="radio"
+            class="custom-control-input"
+        />
+        <label [attr.for]="!(item.optionItem.disabled || disabled) ? item.id : null" class="custom-control-label pr-1"
+            >{{ item.optionItem.label }}
+            <span class="text-muted" *ngIf="item.optionItem.description">{{ item.optionItem.description }}</span>
```

Comment:
Please don't use `text-muted` since it introduces `!important`. Just add the padding to the stylesheet if necessary
```suggestion
            <span *ngIf="item.optionItem.description">{{ item.optionItem.description }}</span>
```